spec/lib/jenkins_pipeline_builder/extensions/wrappers_spec.rb

```diff
@@ -39,6 +39,7 @@
     end
 
     it 'fails parameters are passed' do
+      pending
```

Comment:
Why is this pending?

Reply:
I have added a comment for more context on why this is pending